core/chains/evm/gas/suggested_price_estimator.go

```diff
@@ -19,68 +18,69 @@ import (
 )
 
 var (
-	_ EvmEstimator = &l2SuggestedPriceEstimator{}
+	_ EvmEstimator = &SuggestedPriceEstimator{}
 )
 
 //go:generate mockery --quiet --name rpcClient --output ./mocks/ --case=underscore --structname RPCClient
 type rpcClient interface {
 	CallContext(ctx context.Context, result interface{}, method string, args ...interface{}) error
 }
 
-// l2SuggestedPriceEstimator is an Estimator which uses the L2 suggested gas price from eth_gasPrice.
-type l2SuggestedPriceEstimator struct {
-	services.StateMachine
+// SuggestedPriceEstimator is an Estimator which uses the L2 suggested gas price from eth_gasPrice.
+type SuggestedPriceEstimator struct {
+	utils.StartStopOnce
```

Comment:
I think this may have overwritten another [change](https://github.com/smartcontractkit/chainlink/pull/11117/files#diff-06425d2dc01c25e1e88058a414e9a7a3b189328a57ee94745dc1dac6460a12f5) that went in recently. Think we want to change this back to `services.StateMachine` here.
```suggestion
	services.StateMachine
```